src/components/ScriptureWorkspaceCard.js

```diff
@@ -101,22 +102,22 @@ export default function ScriptureWorkspaceCard({
     title += " (" + id.substr(4) + ")"
   }
 
+  console.log(data)
   return (
     <Card
       title={title}
       classes={classes}
       hideMarkdownToggle={true}
       closeable={true}
       onClose={() => removeBook(id)}
-      key={bookId}
+      key={data.id}
```

Comment:
I'm being picky here. `data` is super confusing as it's the most generic name for a variable... Could we alias it's name in the function argument so that its meaning is more apparent?  I know you didn't create the name and isn't your fault. I do think it's worth fixing here though